### PR TITLE
Fix #340 - Add instructions for Brave

### LINKED ISSUE #340: Please add instructions on how to load the extension on Brave
To the README https://github.com/ipfs/ipfs-companion#install

Bonus points for a gif!

Thank you ❤️

### PR DESCRIPTION
**Pull request by olizilla**
Adds instructions on how to try out the plugin in brave.

See the rendered version at: https://github.com/tableflip/ipfs-companion/blob/cea9db9adf764fbb154186f975df50b6790e1aba/README.md#brave

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +111,26 @@ Try manual installation:
...
+- Add the following to `browser-laptop/app/extensions.js`
+```js
+  // Enable IPFS
+  extensionInfo.setState('ipfs', extensionStates.REGISTERED)
+  loadExtension('ipfs', getExtensionsPath('ipfs'), undefined, 'component')
+```
+https://github.com/ipfs-shipyard/browser-laptop/blob/66f38870fced0dbc55aae7fe1ed905bff602f88e/app/extensions.js#L500-L502
+- In the `browser-laptop` project run `npm install` then in separate shells run `npm run watch` and `npm start`. If you have any trouble running Brave from source, check: https://github.com/brave/browser-laptop#installation
+
+Brave will start up and you should see a badge with your number of connected ipfs peers next to the brave button, top right. (_the ipfs-companion badge [currently doesn't appear]()_). Click on the badge and update your gateway settings to use port `9090`, then go share something with your peers...
```
Comment (lidel) [inline README.md]: the "currently doesn't appear" is an empty link – ? :)

**Comment by alanshaw** [comment]
Needs a bit about removing `browser` property from package.json to enable js-ipfs toggle.

**Comment by olizilla** [comment]
@lidel @alanshaw thanks! I've updated with links to the missing icon issue and pr, and a note on enabling js-ipfs

**Comment by lidel** [comment]
Thank you!

**Pull request merged**